forge-game/src/main/java/forge/game/card/CardFactoryUtil.java

```diff
@@ -556,7 +556,7 @@ public static List<String> sharedKeywords(final Iterable<String> kw, final Strin
                 } else if (k.startsWith("Trample")) {
                     tramplekw.add(k);
                 }
-                allkw.add(k);
+                allkw.add(k.toLowerCase());
```

Comment:
i don't like this one, are you sure that doesn't bite us in the butt?

Reply:
I'd say KW equalness should't depend on the case?
I mean as long as we generate instances using `ignoreCase` it certainly feels weird to categorize them with it now, so I wanted to future-proof against a hard to notice bug.
(+ we still return the unchanged KW from the script param)

Reply:
in very most of the cases, but for some specific Protection or Hexproof it could be problematic when the kw changes into lowercase

Reply:
would it be better if `allkw.add(k.toLowerCase());` was inside an `else` ?

Reply:
Sure, that should make it slightly cleaner